client/galaxy/scripts/viz/sweepster.js

```diff
@@ -737,48 +738,48 @@ var SweepsterVisualizationView = Backbone.View.extend({
             [
                 {
                     key: "name",
-                    label: "Name",
+                    label:  _l("Name"),
```

Comment:
same as with labels above

Reply:
Ok I will remove all this "_l()" , remove all in qunit files and labels because after what you say plenty of them should not be translated.